local_sales/app/views/customers/_featured_products.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1 class="text-4xl font-extrabold text-gray-900 sm:text-3xl">Featured products</h1>
+<div class="mt-6 grid sm:grid-cols-2 sm:gap-x-8 xl:grid-cols-3 xl:gap-x-12">
+   <% @featured_products.each do |featured_product| %>
+     <%= render 'product_card', product: featured_product %>
+   <% end %>
+  </div>
+</div>
```

Comment:
This div, where does it open?

Reply:
This PR is WIP, so we can check all details before assign the *mentor review required* label